src/appleseed/foundation/meta/tests/test_zip.cpp

```diff
@@ -96,13 +96,13 @@ TEST_SUITE(Foundation_Utility_Zip)
             zip(TargetZip, InitialDirectory);
             unzip(TargetZip, TargetDirectory);
 
-            const set<string> ExpectedFiles = recursive_ls(InitialDirectory);
-            const set<string> ActualFiles = recursive_ls(TargetDirectory);
+            const set<string> expected_files = recursive_ls(InitialDirectory);
+            const set<string> actual_files = recursive_ls(TargetDirectory);
 
-            ASSERT_EQ(ActualFiles.size(), ExpectedFiles.size());
+            ASSERT_EQ(actual_files.size(), expected_files.size());
```

Comment:
The arguments should be swapped (the expected value must come first). It's important for error messages, if/when the assertion fails.